docs/workshops/20220920-intro-to-git-and-github/lesson.md

```diff
@@ -0,0 +1,611 @@
+4. On the "Create a new repository" page, choose a name for your repository.
+   For this workshop, name the repository `2022-git-workshop`. 
+   A repository can have any valid directory name, but putting the year at the beginning is a good practice because it makes it clear when the repo was created.
+5. Check the box "Add a README file".
+6. Click the green "Create repository" button at the bottom of the page.
+
+This will create a new repository and take you to the repository's landing page on GitHub.
+
+#### Cloning a repository from remote to local
+
+**Cloning** is the process of copying an existing Git repository from a **remote** location (here, on GitHub) to your **local** computer (here, on our binder instances).
+To clone the repository, click the green "Code" button in the top right hand corner of the repository screen. 
+This creates a drop down menu with clone options.
+We'll select the SSH tab because we configured an ssh key pair.
+Once you select the tab, copy the path that starts with `git@gitub.com:`.
+Then, navigate to your binder instance and use the command below to clone the repository.
+Remember to substitute out the username/URL with your own URL that we copied.
+
+```
+cd ~
+git clone git@github.com:your_username/2022-git-workshop
+cd 2022-git-workshop
+```
+
+#### The basic Git workflow
+
+
+<center>
+<figure markdown>
+  ![Image title](allison_horst_git_overview_small.png){ width="600" }
+  <figcaption> Figure by Allison Horst. <a href='https://twitter.com/allison_horst/status/1523744022421860352' target='_blank'>www.twitter.com/allison_horst</a> </figcaption>
+</figure>
+</center>
+
+We can now make changes to our files in our local repository.
+The basic Git workflow begins when you communicate to Git about changes you've made to your files.
+Once you like a set of changes you’ve made you can tell Git about them by using the command **`git add`**.
+This stages the changes you have made.
+Next, you bake them into the branch using **`git commit`**.
+When you’re ready, you can communicate those changes back to GitHub using **`git push`**.
+This will push your the changes that are in your local repository up to the remote repository.
+
+Let's try this workflow out.
+Throughout this process, we'll use the command `git status` to track our progress through the workflow.
+`git status` displays the state of a working direcotry and the staging area.
+
+```
+git status
+```
+
+We'll use the `echo` command to create a new file, `notes.txt`.
+
+```
+ls
+echo "some interesting notes" > notes.txt
+ls
+```
+
+Take a look at the contents of your `notes.txt` file:
+```
+less notes.txt
+```
+
+And run `git status` to see how creating a new file changes the output of that command:
+```
+git status
+```
+
+Once you have made changes in your repository, you need to tell Git to start tracking the file. 
+The command `git add` adds the files to the "staging area", meaning Git is now tracking the changes.
+
+```
+git add notes.txt
```

Comment:
Ya i see how that could be useful after running `git status`, but I'd rather not teach it at first -- I think it could lead to people accidentally forgetting the sequence to run things, and then lead to files accidentally getting added to repos (e.g. like `.DS_store`). Maybe a month or so after this workshop we could do an AUG on efficiency with git and github if we see an uptick in the number of people committing to Arcadia-Science org?